dodola/tests/test_core.py

```diff
@@ -91,6 +93,52 @@ def test_train_quantiledeltamapping_quantiles_excludeendpoints(variable_kind):
     assert 0.0 not in qdm.ds.hist_q
 
 
+def simple_dtr_factory():
+    return xr.Dataset(
+        {
+            "dtr": xr.DataArray(
+                (np.random.random_sample(size=(12, 24, 10)) + 1e-6) * 50,
+                dims=["lat", "lon", "time"],
+                coords=[
+                    np.arange(-82.5, 90, 15),
+                    np.arange(-172.5, 180, 15),
+                    pd.date_range("2015-01-01", periods=10, freq="D"),
```

Comment:
Could we do something like this:

```suggestion
                    xr.cftime_range("2015-01-01", periods=10, freq="D"),
```

If we change to xr.cftime_range, and then we can drop the pandas import (which we don't have as a direct dependency).

Reply:
thanks! done